litmus-portal/frontend/.stignore

```diff
@@ -0,0 +1,56 @@
+.git
```

Comment:
Rename the file to `.gitignore`

Reply:
This is Okteto's sync config file.. it has to be present. @rajdas98